src/updateTree.js

```diff
@@ -66,18 +66,29 @@ export function updateCurrValue(frameRate, currValue, currVelocity, endValue, k,
     return ret;
   }
   if (Array.isArray(endValue)) {
-    return endValue.map((_, i) => updateCurrValue(frameRate, currValue[i], currVelocity[i], endValue[i], k, b));
+    return endValue.map((_, i) => _updateCurrValue(frameRate, currValue[i], currVelocity[i], endValue[i], k, b));
   }
   if (isPlainObject(endValue)) {
     return Object.keys(endValue).reduce((ret, key) => {
-      ret[key] = updateCurrValue(frameRate, currValue[key], currVelocity[key], endValue[key], k, b);
+      ret[key] = _updateCurrValue(frameRate, currValue[key], currVelocity[key], endValue[key], k, b);
       return ret;
     }, {});
   }
   return endValue;
 }
 
-export function updateCurrVelocity(frameRate, currValue, currVelocity, endValue, k, b) {
+export function updateCurrValue(frameRate, currValue, currVelocity, endValue, k, b) {
+  if (typeof endValue === 'number') {
+    const _k = k || presets.noWobble[0];
```

Comment:
This logic is wrong. If either k or b is null we should return the default. We don't fill a value randomly if only one of them is absent.